module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -82,6 +91,18 @@ public function __construct($baseDir, $safeTheme)
         $this->currentTheme = $this->safeTheme = $safeTheme;
     }
 
+    /**
+     * Provide cache and activate info caching
+     *
+     * @param StorageInterface $cache cache object
+     *
+     * @return void
+     */
+    public function setCache(StorageInterface $cache)
```

Comment:
In order to keep our coverage up, I think we should add some new tests to [ThemeInfoTest](https://github.com/vufind-org/vufind/blob/dev/module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php) to inject mock cache objects to confirm that values coming from the cache get returned and that the cache gets updated when it returns null.